openshift/cirras-underwriting-api-deployment.yaml

```diff
@@ -118,6 +118,15 @@ spec:
                 configMapKeyRef:
                   name: cirras-underwriting-api-config-#{ENV}#
                   key: LOG_LEVEL
+          livenessProbe:
+            httpGet:
+              scheme: HTTP
+              path: /cirras-underwriting-api/v1/checkHealth?callstack=test
+              port: #{TARGET_PORT}#
```

Comment:
I believe TARGET_PORT was renamed to TOMCAT_PORT.

Reply:
Thank you, resolved